docs/installation.rst

```diff
@@ -163,6 +163,23 @@ FUSE for OS X, which is available as a pre-release_.
 
 .. _pre-release: https://github.com/osxfuse/osxfuse/releases
 
+
+FreeBSD
+++++++++
+
+If you're using FreeBSD, you will need to install a package and load the fuse kernel module:
+
+::
+
+    $ sudo pkg install fusefs-libs
+    $ sudo kldload fuse
+    $ borg mount /mnt/backup::root-2013-08-02 /tmp/mymountpoint
+    $ ls /tmp/mymountpoint
+    bin  boot  etc  lib  lib64  mnt  opt  root  sbin  srv  usr  var
+    $ umount /tmp/mymountpoint
+
+
+
```

Comment:
I needed some more stuff to get it installed on freebsd:
https://github.com/borgbackup/borg/blob/master/Vagrantfile#L75

I am not a freebsd user, i just got it working on vagrant for testing.
Some of that stuff is only needed for vagrant or building python and should not be included here.
But some of the other stuff seems missing here (e.g. one does not want to kldload all the time after each reboot).

Also, do not give a mount / umount example here, that is not related to installation.